### DISCUSSION TITLE
babel/packages/babel-helper-create-class-features-plugin verification is waiting for approval

### DISCUSSION THREAD

**Discussion by dhiraj05** [babel/babel#14052] [Releases] [answered]
not able to install babel/packages/babel-helper-create-class-features-plugin currently, as it's waiting for osme CI test for past 5 hrs. Please look into it.

**Comment by JLHwung** [comment, marked as answer]
The package of version 7.16.5 is published. https://www.npmjs.com/package/@babel/helper-create-class-features-plugin

Please see https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67.
